modules/twinklexfd.js

```diff
@@ -718,11 +718,8 @@ Twinkle.xfd.callback.change_category = function twinklexfdCallbackChangeCategory
 									label: 'Opt out of discussion if the request is contested',
 									value: 'rmtr-discuss',
 									name: 'rmtr-discuss',
-									tooltip: 'Use this option when when you are filing an uncontroversial technical request AND you prefer to withdraw the request if it is contested, rather than discuss it. This will suppress the "discuss" link which may be used to convert your request to a discussion on the talk page of the page which is requested to be moved.',
-									checked: false,
-									event: function() {
-										form.newname.required = this.checked;
-									}
+									tooltip: 'Use this option when you prefer to withdraw the request if it is contested, rather than discuss it. This will suppress the "discuss" link, which may be used to convert your request to a discussion on the talk page of the page which is requested to be moved.',
```

Comment:
```suggestion
									tooltip: 'Use this option if you prefer to withdraw the request if contested, rather than discuss it. This suppresses the "discuss" link, which may be used to convert your request to a discussion on the talk page.',
```

One last suggested tweak to reduce the verbosity here, after which this should be ready to merge.